Tests.Integration.Presentation.Web/UI_Configuration/UIConfigurationTests.cs

```diff
@@ -51,11 +47,103 @@ public async Task Can_Get()
             using var response = await UIConfigurationHelper.SendGetRequestAsync(TestEnvironment.DefaultOrganizationId, module);
 
             //Assert
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(HttpStatusCode.NotFound ,response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Get_Returns_Forbidden_If_Not_Allowed_To_Read()
+        {
+            //Arrange
+            var module = A<string>();
+            var (cookie, organization) = await CreatePrerequisitesAsync();
+            var (cookieUser2, _) = await CreatePrerequisitesAsync();
+
+            await UIConfigurationHelper.CreateUIModuleAndSaveAsync(organization.Id, module, cookie);
 
-            var data = await response.ReadResponseBodyAsKitosApiResponseAsync<UIModuleCustomizationDTO>();
-            Assert.Empty(data.Nodes);
-            Assert.NotNull(data.Nodes);
+            //Act
+            using var response = await UIConfigurationHelper.SendGetRequestAsync(organization.Id, module, cookieUser2);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.Forbidden ,response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Put_Deletes_Nodes()
+        {
+            var module = A<string>();
+            var multipleUiCustomizations= UIConfigurationHelper.PrepareTestUiModuleCustomizationDto(10);
+            var singleUiCustomization = UIConfigurationHelper.PrepareTestUiModuleCustomizationDto();
+            
+            using var firstPutResponse = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, multipleUiCustomizations);
+            Assert.Equal(HttpStatusCode.NoContent, firstPutResponse.StatusCode);
+
+            using var secondPutResponse = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, singleUiCustomization);
+            Assert.Equal(HttpStatusCode.NoContent, secondPutResponse.StatusCode);
+
+            var response = await UIConfigurationHelper.GetCustomizationByModuleAsync(TestEnvironment.DefaultOrganizationId, module);
+            Assert.Equal(singleUiCustomization.Nodes.Count(), response.Nodes.Count());
```

Comment:
count is not enough. You want to check if the changes you requested were actually persisted (key + enbaled state) and returned to you